### PR TITLE
Only refresh if internet exists [stage-2]

### PR DESCRIPTION
**Pull request by stulees**
- Updating to use latest widget-common and `hasInternetConnection` utility function, now only refreshing page if internet connection exists
- Added test coverage for refresh and included missing interactivity integration test

### CODE REVIEW

**Comment by stulees** [comment]
@rodrigopavezi Please review, thanks

Presentation - http://rva.risevision.com/#/PRESENTATION_MANAGE/id=391b3b42-87ea-4f82-a8ff-0278fea04cb6?cid=30007b45-3df0-4c7b-9f7f-7d8ce6443013

File: src/widget/webpage.js [hunk 1]
```diff
@@ -145,8 +146,11 @@ RiseVision.WebPage = (function (document, gadgets) {
 
   function _startRefreshInterval() {
     _intervalId = setInterval(function () {
-      // call public function for integration testing purposes
-      _loadFrame();
+      _utils.hasInternetConnection("img/transparent.png", function (hasInternet) {
```
Comment (rodrigopavezi) [inline src/widget/webpage.js]: @stulees I was thinking to check if the page is accessible. What do you think? cheers

Comment (stulees) [inline src/widget/webpage.js]: @rodrigopavezi That is different to a general check for internet connection, what's the advantage? Would it require our proxy server?

Comment (rodrigopavezi) [inline src/widget/webpage.js]: @stulees I was thinking that if the page is unavailable we also don't do a refresh and keep what is there. I don't know, maybe we don't need to go that far.

Comment (stulees) [inline src/widget/webpage.js]: @rodrigopavezi Yeah, I'm concerned about the kinds of URLs being entered and trying to make a HEAD request with it, sometimes they aren't directly targeting an HTML file. Using our own image on our own server provides a definitive check for internet.

Comment (rodrigopavezi) [inline src/widget/webpage.js]: @stulees alright, I think it is good for now. We can change later if that scenario is required. cheers

**Comment by rodrigopavezi** [comment]
@stulees looks good :+1: cheers

**Pull request merged**